integration-tests/tests/src/tests/sync/open-behavior.ts

```diff
@@ -541,7 +541,8 @@ describe("OpenBehaviour", function () {
       await openPromise2;
       throw new Error("openPromise2 should have been rejected..");
     } catch (err: any) {
-      expect(err.message).equals("Operation canceled");
+      //platforms either return "Operation canceled" or "Operation Canceled"
+      expect((err.message as string).toLowerCase()).equals("operation canceled");
```

Comment:
Would it be easy to find the place this differs and solve it or is this thrown by the runtime?

Reply:
Can't find this anywhere in the project so my assumption is that it's thrown by the runtime, especially since the message varies

Reply:
I was able to find some generic error codes within the `boost` library in the `ios/Pods`, which contained an "Operation canceled" string.  Seems the message is out of our hands and is indeed platform specific.